packages/compass-collection/src/components/mock-data-generator-modal/script-screen.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import {
+  Body,
+  Code,
+  Copyable,
+  css,
+  cx,
+  InlineCode,
+  Language,
+  Link,
+  Overline,
+  palette,
+  spacing,
+  useDarkMode,
+} from '@mongodb-js/compass-components';
+import { useConnectionInfo } from '@mongodb-js/compass-connections/provider';
+
+const RUN_SCRIPT_COMMAND = `
+mongosh "mongodb+srv://<your-cluster>.mongodb.net/mockDataDB" \\
```

Comment:
nit: `mockDataDB` should be replace with a <your-database>
supernit: the user placeholders should all either be camelCase or hyphen-separated